README.md

```diff
@@ -138,6 +138,7 @@ jsdom.env(config);
 - `config.pool`: an object describing which agents to use for the requests; defaults to `{ maxSockets: 6 }`, see [request module](https://github.com/request/request#requestoptions-callback) for more details.
 - `config.agentOptions`: the agent options; defaults to `{ keepAlive: true, keepAliveMsecs: 115000 }`, see [http api](https://nodejs.org/api/http.html) for more details.
 - `config.strictSSL`: if `true`, requires SSL certificates be valid; defaults to `true`, see [request module](https://github.com/request/request#requestoptions-callback) for more details.
+- `config.encoding`: the default encoding; defaults to `UTF-8`
```

Comment:
We should specify that this is an _encoding label_ from https://encoding.spec.whatwg.org/#names-and-labels, and make sure we normalize all labels to their corresponding encoding using some sort of lookup table.

We should also list somewhere (maybe in the readme, or maybe in a wiki page if it's too long) which encodings we do and don't support from that list.